### PR TITLE
Add auxiliary logging for socks5

### LINKED ISSUE #100: Logging of auxiliary data for socks5.

### PR DESCRIPTION
**Pull request by realsdx**
Fixes #100

### CODE REVIEW

File: heralding/capabilities/socks5.py [hunk 1]
```diff
@@ -63,6 +64,23 @@ class Socks5(HandlerBase):
         else:
             logger.debug("Wrong authentication data: %r" % auth_data)
 
+    def get_auxiliary_data(self, authmethods):
```
Comment (johnnykv) [inline heralding/capabilities/socks5.py]: Could we also log the socks5 version? As far as i read the code, the client also transmits a value containing  some kind of versioning?

Comment (realsdx) [reply heralding/capabilities/socks5.py]: The client transmits the Socks version. Here it should be always 5. If, its not 
the line 65 executes.

**Comment by kajojify** [comment]
Thank you for the PR. The PR looks good. What do you think about adding a condition for logging
_NO ACCEPTABLE METHODS_?

**Comment by realsdx** [comment]
I think it should be added as its in RFC. but, why even a client will transmit that in first place?  if it doesn't want to connect it could simply refuse any connection

**Comment by realsdx** [comment]
Updated it, please review.

**Pull request merged**